tests/integration_tests.rs

```diff
@@ -747,6 +747,25 @@ fn config_location_when_generating() {
     assert!(tmp_config_path.exists());
 }
 
+#[test]
```

Comment:
Can you please add the new env. variable to `bat_raw_command_with_config` to make sure it is *removed from the environment* for all integration tests? This is important in order to make sure that tests pass, even in environments where a user has set some env. variables.